base/abstractdict.jl

```diff
@@ -335,6 +335,10 @@ function _typeddict(d::AbstractDict, others::AbstractDict...)
     Dict{K,V}(d)
 end
 
+reduce(m::typeof(merge), items::AbstractVector) = _merge!(mapreduce(typeof, typejoin, items), items)
```

Comment:
`m` isn't a typical name for this argument, and it's not needed at all.

Reply:
Agreed.